templates/primary-secondary/primary-secondary.js

```diff
@@ -1053,6 +1077,11 @@ class TemplatePrimarySecondary extends LocalizeCoreElement(LitElement) {
 				</footer>
 			</div>
 		`;
+
+		if (this.hasForm) return html`<d2l-form hide-error-summary @d2l-form-invalid=${this.#handleInvalidForm} @d2l-form-submit=${this.#handleFormSubmit} @d2l-form-dirty=${this.#handleFormDirty}>
```

Comment:
Another option here: give the form a reference to the `id` of the existing summary, i.e. `<d2l-form summary-id="some-id">`. Then it would set the errors automatically on that summary element and know not to create its own. This can work since their both in the same light DOM.